BraveRewardsUI/Welcome/WelcomeViewController.swift

```diff
@@ -67,4 +71,17 @@ class WelcomeViewController: UIViewController {
       })
     }
   }
+  
+  private func tappedDisclaimerLink(_ url: URL) {
+    switch url.path {
+    case "/terms":
+      state.delegate?.loadNewTabWithURL(URL(string: "https://brave.com/terms-of-use/")!) //swiftlint:disable:this force_unwrapping
```

Comment:
We may want to put make these URLs static and use them in multiple places in case the URL ever changes